src/temp/CarbonCapture/CarbonCapture.tsx

```diff
@@ -0,0 +1,179 @@
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import React, { useState } from 'react';
+import style from './CarbonCapture.module.scss';
+import DownArrow from '../icons/DownArrow';
+import { Trans, useTranslation } from 'next-i18next';
+import UpArrow from '../icons/UpArrow';
+import { CO2BarGraph, CO2CaptureData } from './BarGraph';
+import NewInfoIcon from '../icons/NewInfoIcon';
+
+interface TabPanelProps {
+  index: number;
+  value: number;
+  beforeIntervation: number;
+  byProject: number;
+  sitePotential: number;
+}
+
+type carbonCapture = Omit<TabPanelProps, 'value'>;
+
+function CustomTabPanel(props: TabPanelProps) {
+  const {
+    value,
+    index,
+    beforeIntervation,
+    byProject,
+    sitePotential,
+    ...other
+  } = props;
+
+  const { t } = useTranslation(['projectDetails']);
+  const [isExtend, setIsExtend] = useState(false);
+
+  const carbonCaptureDataProps = {
+    beforeIntervation,
+    byProject,
+    sitePotential,
+  };
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      className={style.carbonCaptureMainContainer}
+      {...other}
+    >
+      {value === index && (
+        <>
+          <div className={style.carbonCaptureInfoContainer}>
+            <div className={style.carbonCaptureLabelMainContainer}>
+              <div className={style.carbonCaptureLabelContainer}>
+                <div className={style.carbonCapture}>
+                  <Trans i18nKey="totalCO₂Captured">
+                    Total CO₂ Captured <p>(tons)</p>
```

Comment:
Add translations here, add separate translation for `Total CO₂ Captured` and `tons`